src/fxcorrelator_xengops.py

```diff
@@ -189,17 +189,22 @@ def xeng_vacc_sync(corr, vacc_load_time=None):
     Assumes that the x-engines are all receiving data.
     :return:
     """
+    min_ld_time=1
     if vacc_load_time is None:
-        unix_time_diff = 2
-    else:
-        time_now = time.time()
-        if vacc_load_time < time.time() + 1:
-            raise RuntimeError('Load time of %.4f makes no sense at current time %.4f' %
-                               (vacc_load_time, time_now))
-        unix_time_diff = vacc_load_time - time.time()
-    wait_time = unix_time_diff + 3
+        corr.logger.info("Vacc sync time not specified. Syncing in ~2 seconds' time.")
+        vacc_load_time = time.time()+2
+
+    t_now=time.time()
+    if vacc_load_time < t_now + min_ld_time:
+        raise RuntimeError('Cannot load at a time in the past. '
+                           'Need at least %2.2f seconds lead time. You asked for %s.%i, and it is now %s.%i.' %(\
```

Comment:
I think there is a syntax error here, since the parameters to the string interpolation need to be in brackets. Also, since everything is already in parenthesis, there is no need for explicit line continuations ('\'.)  So this should be something like:

'''
.... " % (min_ld_time, 
    time.strftime(' ...'),
    ...) )